diff --git a/dist/declarations/src/three-types.d.ts b/dist/declarations/src/three-types.d.ts
index 64ff8491be197d83b3279f48407fec78eebd5c84..b5e58de52771fad6894d2c6e37721b2f683cea8e 100644
--- a/dist/declarations/src/three-types.d.ts
+++ b/dist/declarations/src/three-types.d.ts
@@ -21,7 +21,7 @@ export interface NodeProps<T, P> {
     attach?: AttachType;
     args?: Args<P>;
     children?: React.ReactNode;
-    ref?: React.RefCallback<T> | React.RefObject<React.ReactNode> | null;
+    ref?: React.Ref<T> | React.MutableRefObject<T | undefined>;
     key?: React.Key;
     onUpdate?: (self: T) => void;
 }
